conan/tools/cmake/cmakedeps/templates/targets.py

```diff
@@ -58,6 +58,10 @@ def template(self):
         get_filename_component(_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
         file(GLOB DATA_FILES "{{data_pattern}}")
 
+        if(NOT DEFINED {{ file_name }}_MESSAGE_MODE)
```

Comment:
Is it OK to assume the code generated by config..py is included before this? Or should I replicate the `if({{ file_name }}_FIND_QUIETLY)` check here?

Reply:
Yes, it is safe, this file will always be included from xxx-config.cmake, if that is what you mean, so ``{{ file_name }}_MESSAGE_MODE`` should already be defined at this point.